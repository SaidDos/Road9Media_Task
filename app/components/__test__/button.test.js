import 'react-native';
import React from 'react';
import Button from '../buttons/button';

import renderer from 'react-test-renderer';

it('should renders correctly', () => {
  const tree = renderer.create(<Button buttonName={'test'} />).toJSON();
  expect(tree).toMatchSnapshot();
});
