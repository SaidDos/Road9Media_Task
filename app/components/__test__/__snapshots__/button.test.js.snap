// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#07dddd",
      "borderRadius": 10,
      "height": 50,
      "justifyContent": "center",
      "margin": 20,
      "opacity": 1,
      "width": 320,
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "center",
        "color": "black",
        "fontSize": 18,
        "fontWeight": "bold",
      }
    }
  >
    test
  </Text>
</View>
`;
